Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -1294,6 +1298,34 @@ namespace Multiplayer
         }
     }
 
+    void MultiplayerSystemComponent::OnRootSpawnableReady([[maybe_unused]] AZ::Data::Asset<AzFramework::Spawnable> rootSpawnable, [[maybe_unused]] uint32_t generation)
+    {
+        SpawnPlayersWaitingToBeSpawned();
+    }
+
+    void MultiplayerSystemComponent::SpawnPlayersWaitingToBeSpawned()
+    {
+        IMultiplayerSpawner* spawner = AZ::Interface<IMultiplayerSpawner>::Get();
+        if (!spawner)
+        {
+            AZLOG_ERROR("Attempting to spawn players on level load failed. No IMultiplayerSpawner found. Ensure one is registered.");
```

Comment:
Is this from lack of the right components on the level, or from a lack of some Multiplayer gem interfaces?

Reply:
This is from the lack of the right component. NetworkPlayerSpawner components will register themselves on activate. Since the ServerClient calls host before calling load-level, there aren't any NetworkPlayerSpawner components active yet :)